src/components/BackToTop.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <v-btn
+    v-show="model"
+    v-scroll="onScroll"
+    position="fixed"
+    location="bottom right"
+    class="mt-auto pointer-events-initial ma-4"
+    color="surface-2"
+    elevation="8"
+    icon="mdi-chevron-up"
+    size="large"
+    @click="onClick"
+  />
+</template>
+
+<script setup>
+import { ref } from 'vue';
+
+const model = ref(false);
+
+function onScroll() {
+  model.value = window.scrollY > 200;
+}
```

Comment:
Let's rename the ref to `show` so that it's a bit more consistent with how we've named the variables defining whether to render other modal elements